src/validation/services/subject.js

```
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+const Category = require('~/models/category');
+const Subject = require('~/models/subject');
+
+const validateSubjectOnCreate = async (subjectData) => {
+const { name, category } = subjectData;
```

ESLint

[error] Expected indentation of 2 spaces but found 0.

(indent)